db/models/Location.js

```diff
@@ -17,8 +17,8 @@ const locationSchema = new Schema({
     postcode: { type: Number, required: true },
     city: { type: String, required: true },
   },
-  latitude: { type: String, required: false },
-  longitude: { type: String, required: false },
+  coordinates: [{ type: Number, required: false }],
```

Comment:
Does this work properly? My Schema looks like this: 

```javascript
coordinates: {type: [Number], required: false}
```

But maybe both Syntaxes work?!

Reply:
ups...your version should be the right way. but yes it works 😬